node/test/go/dev-mode/test/integration/index.test.js

```diff
@@ -56,15 +56,21 @@ describe('Integration', function () {
         ]),
         config: {
           test: ({ invocationsData }) => {
-            for (const [, reqRes] of invocationsData.map((data) => data.reqRes).entries()) {
-              expect(reqRes.length).to.equal(1);
-              for (const payload of reqRes) {
-                expect(payload).to.haveOwnProperty('region');
-                expect(payload).to.haveOwnProperty('accountId');
-                expect(payload).to.haveOwnProperty('timestamps');
-                expect(payload).to.haveOwnProperty('payloads');
-              }
-            }
+            // Replace with external + sdk integration results once the sdk is configured
+            // to communicate with the external extension
+            const allReqResData = invocationsData.map((data) => data.reqRes);
+            expect(allReqResData[0].length).to.equal(0);
+            expect(allReqResData[1].length).to.equal(0);
+
+            // for (const [, reqRes] of invocationsData.map((data) => data.reqRes).entries()) {
+            //   expect(reqRes.length).to.equal(1);
+            //   for (const payload of reqRes) {
+            //     expect(payload).to.haveOwnProperty('region');
+            //     expect(payload).to.haveOwnProperty('accountId');
+            //     expect(payload).to.haveOwnProperty('timestamps');
+            //     expect(payload).to.haveOwnProperty('payloads');
+            //   }
+            // }
```

Comment:
Why this code was commented out? 

Also if given logic is no longer applicable, let's simply remove it instead of commenting it out

Reply:
Yeah no longer applicable I will remove the comments 👍